docs/modules/github-provider.md

```
@@ -92,13 +93,13 @@ flowchart TD
 create_issue -> set_issue_type -> add_labels (x N) -> add_project_item -> set_project_fields
 ```
 
-The optimized flow sets labels, issue type, and project in a single `createIssue` mutation. Only project field assignment (e.g. Size) requires follow-up calls:
+The optimized flow sets labels, issue type, and project in a single `createIssue` mutation. Only project field assignment (e.g. Size, Priority, or Iteration) requires follow-up calls:
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)